roles/hdfs-standalone/defaults/main.yml

```diff
@@ -3,6 +3,7 @@ hdfs_version: "2.6.0"
 hdfs_conf_dir: "/etc/hadoop/conf"
 hdfs_install_dir: "/usr/local/share"
 hdfs_namenode_port: 8020
+hdfs_namenode_host: ""
```

Comment:
I would prefer use 'localhost' as dummy and change it on higher level if there are needed.
TBD